### ISSUE TITLE
Switching language.

### ISSUE THREAD

**Issue by KolkolAlex** [sds100/KeyMapper#870]
Please tell me how I can switch languages on my physical keyboard.
Now only GUI works. But I used to have hotkey (Shift+Space) in my prev keyboard ("Russian keyboard").
And then typed directly from hardware keyboard on the chosen language.
Please help me with this problem.

**Comment by sds100** [comment]
Hi, which Android version do you have? Which keyboard are you using?

**Comment by KolkolAlex** [comment]
Hello! Sorry for delay. Android 9 (ugoos x3pro) and keyboard is 2.4Hz APE electonics (recognixed as 2.4H Composite device). Besides I plugged USB keyboard (Genius Luxemate 320 (recognized as barcode reader) - no effect (only once it switched but only on gui - not for hardware, and I can't even repeat that). I also assigned new keycode_language_switch (204). But it just doesn't work without any warnings.

**Comment by sds100** [comment]
If you disable Key Mapper by pausing your key maps does that fix the issue?

**Comment by KolkolAlex** [comment]
No. Any changes. There is also an issue - taping on gui key for lang switching doen not switches language for typing on the firmware keyboard...

**Comment by sds100** [comment]
Which input method (on-screen keyboard) are you using?

**Comment by KolkolAlex** [comment]
"KeyMapper GUI Keyboard". And may be it is not able to switch language like "Russian keyboard" does. 
[Russian Keyboard.zip](https://github.com/sds100/KeyMapper/files/7844786/Russian.Keyboard.zip)
There in preferences I can choose hotkey combination for switching lang.
It's just to show what I'm talking about)

**Comment by sds100** [comment]
Key Mapper GUI Keyboard is now deprecated
